### ISSUE TITLE
xpra-5.0.4_1: /usr/lib/python3.12/site-packages/xpra/scripts/parsing.py:327: SyntaxWarning: invalid escape sequence '\w'

### ISSUE THREAD

**Issue by datenwolf** [Xpra-org/xpra#4070]
Installing Xpra through system package manager, at the "Byte-compiling python3.12 code for module xpra..." stage the following errors are reported:

```
xpra-5.0.4_1: configuring ...
Byte-compiling python3.12 code for module xpra...
usr/lib/python3.12/site-packages/xpra/scripts/parsing.py:327: SyntaxWarning: invalid escape sequence '\w'
  legacy_ssh = re.search("^ssh:([\w]([\w-]{0,61}[\w])?):([\d]{1,5})$", display_name)
usr/lib/python3.12/site-packages/xpra/scripts/parsing.py:327: SyntaxWarning: invalid escape sequence '\w'
  legacy_ssh = re.search("^ssh:([\w]([\w-]{0,61}[\w])?):([\d]{1,5})$", display_name)
```

**Comment by totaam** [comment]
@datenwolf distro?

**Comment by datenwolf** [comment]
The distro is Voidlinux. However, this issue is independent of that. 

Pasting the string literal here

https://github.com/Xpra-org/xpra/blob/1d84ece83249956a248e374ce818720b4422b4cc/xpra/scripts/parsing.py#L327

into a Python REPL exposes the issue

```
Python 3.12.0 (main, Oct  6 2023, 16:16:58) [GCC 12.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> "^ssh:([\w]([\w-]{0,61}[\w])?):([\d]{1,5})$"
<stdin>:1: SyntaxWarning: invalid escape sequence '\w'
'^ssh:([\\w]([\\w-]{0,61}[\\w])?):([\\d]{1,5})$'
```

I presume that this is supposed to be an `r""` string, so that backslashes that are part of the regex don't need to be escaped.

**Comment by totaam** [comment]
Why are you running a git version from 5 months ago?
This was fixed way back in 3e230cebb82a1e552d176c97dc489b10da062a00

**Comment by totaam** [comment]
My bad, the fix was in git master since 3e230ce
But never made it in the 5.x branch until now. Sorry about that.
